Sources/PrimerSDK/Classes/User Interface/Root/PrimerRootViewController.swift

```diff
@@ -490,7 +490,13 @@ internal class PrimerRootViewController: PrimerViewController {
             return viewController.view.bounds.size.height + nc.navigationBar.bounds.height
         }
     }
+}
+
+extension PrimerRootViewController {
     
+    func presentPrimerTestViewControllerForPaymentMethodType(_ type: PrimerPaymentMethodType) {
```

Comment:
Why is this needed?

Reply:
Not needed - leftover from the implementation I thought it would be good with HUC - then realized not needed